sample-status-updater-application/src/main/java/life/qbic/samplestatusupdater/app/SampleUpdater.java

```diff
@@ -72,18 +74,30 @@ public void execute() {
         SampleTrackingService sampleTrackingService = new SampleTrackingServiceConnector(serviceList.get(0), credentials);
 
         OpenBisSearchService searchService = new SampleSearchConnector(session.getApi(), session.getToken());
-        Date lastSearchDate = parseDateFromStringWithPattern((String) properties.get("lastSearchDate"), "yyyy-MM-dd");
+        Instant lastSearchDate = null;
+        try{
+            lastSearchDate = Instant.parse((String) properties.get("lastSearchDate"));
+        } catch (Exception e) {
+            LOG.error("Could not parse last search date " + properties.get("lastSearchDate"));
+            Arrays.stream(e.getStackTrace()).forEach( LOG::error );
```

Comment:
Is there a reason you do not use `LOG.error(e.getMessage, e)`?

Reply:
ah, i wondered if there is a convenience method available :) thanks